api/config/config_perf.json

```diff
@@ -140,5 +140,7 @@
   "accessRenewal": {
     "expiryDays": 365,
     "expiryDaysWarningThresholds": [ 1, 3, 7, 15, 30 ]
+  },
+  "offlineBatch": {
```

Comment:
This is not a convention we've used before, but I think I prefer to omit this "unsafe" key entirely here, rather than leaving it in as `""`. I think empty string is more ambiguous than in the other cases we have, where the default is simply `false`.

I want to leave this tuple in place, because I intend to use it soon for other configuration values such as the "usersPerTask" control on project audit, and access synchronization.